template/cordova/lib/log.js

```diff
@@ -30,33 +30,87 @@ var platformRoot = path.join(__dirname, '..', '..'),
     configPath   = path.join(projectRoot, 'config.xml');
 
 // variables
-var appTracingInitialState = null,
+var appTracingLog          = 'Microsoft-Windows-AppHost/ApplicationTracing',
+    appTracingInitialState = null,
     appTracingCurrentState = null,
-    adminInitialState = null,
-    adminCurrentState = null,
+    adminLog               = 'Microsoft-Windows-AppHost/Admin',
+    adminInitialState      = null,
+    adminCurrentState      = null,
+    timers                 = [],
     appName;
 
 /*
  * Gets windows AppHost/ApplicationTracing and AppHost/Admin logs
  * and prints them to console
  */
-module.exports.run = function() {
-    getLogState('Microsoft-Windows-AppHost/Admin').then(function (state) {
+module.exports.run = function(args) {
+    var dump = false, 
+        startTime = new Date(new Date().getTime() - 10 * 60 * 1000).toISOString(); // show last 10 minutes by default
+
+    if (args.length > 0) {
+        for (var i = 0; i < args.length; i++) {
+            switch (args[i]) {
```

Comment:
Ideally, we should use a library for this - the one that is used by cordova-windows should be re-used here instead of doing this on your own.